<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellWindow</class>
 <widget class="QWidget" name="SellWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>销售中</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="MainFrame">
     <layout class="QGridLayout" name="gridLayout" columnstretch="3,5,17">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="MenuFrame">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0.3);
border-radius: 15px</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="ret_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:grey;
color: white;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>返回登录界面</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:15px;</string>
         </property>
         <property name="text">
          <string>收银台
您好，销售员</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>120</width>
           <height>631</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollArea{
border: 1px solid;
border-color:rgba(0, 170, 255, 0.5);
background-color:rgba(0, 170, 255, 0.5);
}
QScrollBar::handle:vertical {/*设置滑动条*/
	border: none;
    border-radius:15px; 
	background-color: rgb(55, 156, 212);
  }
QScrollBar::add-line:vertical {
border: 0px solid grey;
background: #32CC99;
height: 0px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
border: 0px solid grey;
background: #32CC99;
height: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>118</width>
            <height>629</height>
           </rect>
          </property>
          <widget class="QPushButton" name="sell_btn">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>90</width>
             <height>90</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:15px;
	border:1px solid rgb(100, 100,189)

}</string>
           </property>
           <property name="text">
            <string>收银主页</string>
           </property>
          </widget>
          <widget class="QPushButton" name="rule_btn">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>90</width>
             <height>90</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:15px;
	border:1px solid rgb(100, 100,189)

}</string>
           </property>
           <property name="text">
            <string>充值规则</string>
           </property>
          </widget>
          <widget class="QPushButton" name="mem_btn">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>300</y>
             <width>90</width>
             <height>90</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:15px;
	border:1px solid rgb(100, 100,189)

}</string>
           </property>
           <property name="text">
            <string>会员信息</string>
           </property>
          </widget>
          <widget class="QPushButton" name="tip_btn">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>430</y>
             <width>90</width>
             <height>90</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:15px;
	border:1px solid rgb(100, 100,189)

}</string>
           </property>
           <property name="text">
            <string>订单信息</string>
           </property>
          </widget>
          <widget class="QPushButton" name="sns_btn">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>540</y>
             <width>90</width>
             <height>90</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:15px;
	border:1px solid rgb(100, 100,189)

}</string>
           </property>
           <property name="text">
            <string>发送短信</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="MarketFrame">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0.3);
border-radius: 15px</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="Logined">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:15px;
border:1px solid rgb(100, 100,189)</string>
         </property>
         <property name="text">
          <string>当前会员：未选</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="MemberRegist">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #57bd6a;
color: #f9ffff;
border-radius: 10px;
}
QPushButton:pressed {
background-color: #4eaa5f;
}</string>
         </property>
         <property name="text">
          <string>会员注册</string>
         </property>
        </widget>
        <widget class="QPushButton" name="MemberLogin">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #57bd6a;
color: #f9ffff;
border-radius: 10px;
}
QPushButton:pressed {
background-color: #4eaa5f;
}</string>
         </property>
         <property name="text">
          <string>选择会员</string>
         </property>
        </widget>
        <widget class="QLabel" name="FinalPrice">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>620</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:15px;
border:1px solid rgb(100, 100,189)</string>
         </property>
         <property name="text">
          <string>总金额：0元</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="GachaButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>710</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #ffaa00;
color: #f9ffff;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(234, 156, 0);
}</string>
         </property>
         <property name="text">
          <string>结算</string>
         </property>
        </widget>
        <widget class="QFrame" name="MarketList">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>160</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
border-radius: 15px;
background-color:transparent;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="ClearButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>710</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: #ffaa00;
color: #f9ffff;
border-radius: 10px;
}
QPushButton:pressed {
background-color: rgb(234, 156, 0);
}</string>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="ProductFrame">
        <property name="styleSheet">
         <string notr="true">QFrame
{
	background-color: rgba(255, 255, 255, 0.3);
	border-radius: 15px
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QSlider" name="MySlider">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>160</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
            border: none;
            height: 2px;
            background-color: lightgray;
        }

        QSlider::handle:horizontal {
            background-color: #f1964c;
            width: 16px;
            margin: -8px 0px -8px 0px;
            border-radius: 8px;
        }

        QSlider::sub-page:horizontal {
            background-color: #f1964c;
        }</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QFrame" name="ProductsList">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>280</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border-radius: 15px;
	background-color:transparent;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="MySearch">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>110</y>
           <width>113</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
            border: 1px solid lightgray;
            border-radius: 15px;
            padding-left: 5px;
            padding-right: 60px;
            background-color: rgba(255, 255, 255, 0.8);
        }</string>
         </property>
         <property name="placeholderText">
          <string>键入信息以筛选商品...</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="Radio1">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>90</y>
           <width>86</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>按名称</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="Radio2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>130</y>
           <width>86</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>按价格</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>270</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
